src/modules/sub-items/components/pagination-info/pagination.info.component.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './css/pagination.info.component.css';
+
+const PaginationInfoComponent = ({ totalCount, viewingCount }) => {
```

Comment:
I'm not sure we even need this component as this will be only one `div` if you follow the comment below.

Reply:
We can make it with one `div` - I will delete this component and move code to sub.items.module.js.